CartPole/__init__.py

```diff
@@ -54,21 +54,24 @@
 
 # endregion
 
-PATH_TO_CONTROLLERS = './Controllers/'  # Path where controllers are stored
-PATH_TO_EXPERIMENT_RECORDINGS = './ExperimentRecordings/'  # Path where the experiments data is stored
+import yaml
+config = yaml.load(open("config.yml", "r"), Loader=yaml.FullLoader)
+
+PATH_TO_CONTROLLERS = config["cartpole"]["PATH_TO_CONTROLLERS"]
+PATH_TO_EXPERIMENT_RECORDINGS = config["cartpole"]["PATH_TO_EXPERIMENT_RECORDINGS"]
 
 
 class CartPole:
 
-    def __init__(self):
+    def __init__(self, initial_state=s0):
```

Comment:
Optional constructor is now used with GUI: In GUI, you can now select initial position and angle. The corresponding state is passed to this constructor **by reference** (because it's a numpy array). Therefore, if the sliders are changed, this state changes with it!